### PR TITLE
samples: cellular: nrf_cloud: Control online alert

### PR DESCRIPTION
**Pull request by plskeggs**
By default, no longer send an alert on startup.

Add logging of the reset reason, and if the online alert is enabled, include that in the value.

IRIS-9688

### CODE REVIEW

File: samples/cellular/nrf_cloud_multi_service/README.rst [hunk 1]
```diff
@@ -122,8 +122,11 @@ Application thread and main application loop
 ============================================
 
 The application thread is implemented in the :file:`src/application.c` file, and is responsible for the high-level behavior of this sample.
-It performs the following major tasks:
 
+When it starts, it logs the `reset reason code <nRF9160 RESETREAS_>`_.
+If :kconfig:option:`CONFIG_SEND_ONLINE_ALERT` is enabled, it sends an alert to nRF Cloud containing the reset reason as the value field.
```
Comment (peknis) [inline samples/cellular/nrf_cloud_multi_service/README.rst:127]:
```suggestion
If the :kconfig:option:`CONFIG_SEND_ONLINE_ALERT` Kconfig option is enabled, it sends an alert to nRF Cloud containing the reset reason as the value field.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by glarsennordic — APPROVED** [review]

**Comment by glarsennordic** [comment]
That's interesting, did me reviewing this clear the review request for you @tony-le-24 @jayteemo ?

File: samples/cellular/nrf_cloud_multi_service/src/application.c [hunk 2]
```diff
@@ -321,8 +322,29 @@ static void test_counter_send(void)
 	}
 }
 
+static void report_reset(void)
```
Comment (jayteemo) [inline samples/cellular/nrf_cloud_multi_service/src/application.c:325]: nit
```suggestion
static void print_reset_reason(void)
```

**Review by jayteemo — APPROVED** [review]

**Comment by tony-le-24** [comment]
@glarsennordic it does clear my review request.

**Review by tony-le-24 — APPROVED** [review]
LGTM, green after manually CI testing.

**Comment by tony-le-24** [comment]
There are compliance issues about some trailing white spaces.

**Review by peknis — APPROVED** [review]

**Pull request merged**